data/PYUSD/data.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "PayPal USD",
+  "symbol": "PYUSD",
+  "decimals": 6,
```

Comment:
hey @skellet0r let me know if you need any help updating this with the new factory